src/components/scenes/Loans/LoanCreateScene.tsx

```diff
@@ -64,6 +65,12 @@ export const LoanCreateScene = (props: Props) => {
   const existingLoanAccount = useSelector(state => state.loanManager.loanAccounts[borrowEngineWallet.id])
   if (existingProgramId != null) navigation.navigate('loanStatus', { actionQueueId: existingProgramId, loanAccountId: existingLoanAccount.id })
 
+  // Force enable tokens required for loan
+  useAsyncEffect(async () => {
```

Comment:
That was the first place I tried to add it, but it didn't seem to be soon enough.

Reply:
We need the exchange rates sooner because we rely on the FiatAmountInputCard to give us a native borrow amount.

All these hacks can be deprecated after https://app.asana.com/0/1200382638405084/1203374649830369/f